AUTHORING_GUIDE.md

```diff
@@ -428,7 +428,51 @@ def test_resource():
     ...
 ```
 
-### Running tests
+### Test Environment Setup
+
+Because all tests are system tests that use live resources, running tests
+requires a Google Cloud project with billing enabled, as covered under
+[Creating and Managing Projects](https://cloud.google.com/resource-manager/docs/creating-managing-projects).
+
+Once you have your project created and configured, you'll need to set environment
+variables to identify the project and resources to be used by tests. See
+[testing/test-env.tmpl.sh](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/testing/test-env.tmpl.sh)
+for a list of all environment variables used by all tests. Not every test
```

Comment:
Can we add something to the effect of "All required environment variables should be listed in the README. If you find one is missing, please add instructions for setting it as part of your PR."

Reply:
Done